### DISCUSSION TITLE
关于播放器的音频响度均衡&其他

### DISCUSSION THREAD

**Discussion by NCXH945** [the1812/Bilibili-Evolved#4597] [功能建议 / Ideas]
刚刚在播放av538418888的时候发现一个问题，把视频的音频源扒下来在本地播放时，同音量下响度的表现与它在网页端不太一样：
![95a6ade7cbf71f9b64db4e1eb6c80fa3](https://github.com/the1812/Bilibili-Evolved/assets/100441737/ca626a9d-2b7f-4ff4-9228-83cbda61a6d0)
我个人猜测这是启用了响度均衡，但播放器内找不到任何相关的设置
并且，在播放器内点击右键-视频音频调节以调出均衡器，可以在短暂的空档后使声音变得更正常——我不知道如何形容，大概是动态范围变大，中低频听起来更饱满——即使你没有对均衡器的参数做出任何改变(这个问题已经断断续续存在两年有余)
可不可以做一个组件来拦截播放器对音频的“添油加醋”？

**Comment by the1812** [comment]
> 声音变得更正常

没感觉出有啥区别...

**Reply by lolvlol** [reply]
我倒是更希望加入和手机端一样的音量均衡功能，网页端有的视频音量太大每次都要调，手机端就无感。

**Comment by Tinhone** [comment]
如果把本地音频放进浏览器播放会是什么样子呢...？或许可以测试一下

**Comment by suheandzl** [comment]
我最近也发现类似的问题 我还以为是脚本或者扩展的问题 各种排查也没排查出来 比如BV1iv411j7wL（[官方] 云南山歌 《朝你大胯捏一把》） 在网页加载未完成的时候声音正常 而加载完毕后声音低了一个量级 看到此帖后在网页加载完成的时候打开音频设置随便点击（实际上未修改任何设置）后 声音恢复到载未完成的时候声音量级上了 麻烦作者大大关注一下这个问题
